File: vetor/exercicio5.py

numeros = [0] * 100

for i in range(100):
    numeros[i] = int(input(f"Digite o número {i+1}: "))

soma = 0
for i in range(100):
    soma += numeros[i]

media = soma / 100

print(f"A média aritmética do conjunto é: {media:.2f}")

File: vetor/exercicio5.c
#include <stdio.h>

int main() {
    int vetor[100];
    int soma = 0;

    printf("Digite 100 números inteiros:\n");
    for (int i = 0; i < 100; i++) {
        printf("Número %d: ", i + 1);
        scanf("%d", &vetor[i]);
        soma += vetor[i];
    }

    double media = (double)soma / 100;

    printf("\nMédia aritmética: %.2f\n", media);

    return 0;
}

